panel/README.md

```diff
@@ -0,0 +1,29 @@
+# Panel
+
+## Usage
+
+The Panel component can be used in both XML and Kotlin
+
+Example of a padded panel in XML
+
+```xml
+<net.skyscanner.backpack.panel.bpkpanel
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    app:bpk_padding="true">
+
+    <textview
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:text="panel message" />
+</net.skyscanner.backpack.panel.bpkpanel>
+```
+
+Example of a padded panel in Kotlin
+
+```Kotlin
+import net.skyscanner.backpack.panel
+
+val panel = BpkPanel(context)
```

Comment:
I don't have anything against it, I just don't know why should we do one or the other, is it best practice or something like that?

Reply:
It has the same effect, but since this is an `initialisation` block it looks much nicer to have it inside the apply block. Plus avoid the rewriting the panel variable, so you could just add

```
BpkPanel(context).apply {
   padding = true
}
```

This is actually the only instance where apply is explicitly preferred in Kotlin. E.g,  when you have a function that returns a new object, you don't need to declare a new variable for this.